src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -168,8 +167,8 @@ public static void SetMinMaxThreadsTest_ChangedInDotNetCore()
             try
             {
                 Assert.True(ThreadPool.SetMinThreads(0, 0));
-                VerifyMinThreads(0, 0);
```

Comment:
Should we do VerifyMinThreads(1, 1) instead?

Reply:
I'll go ahead and add that to https://github.com/dotnet/corefx/pull/25061, as that is tied to the same CoreCLR change and both would have to go in afterwards